addons/account/models/account_move.py

```diff
@@ -2569,12 +2573,22 @@ def _autopost_draft_entries(self):
         It is used to post entries such as those created by the module
         account_asset.
         '''
+        try:
+            batch_size = int(self.env['ir.config_parameter'].sudo().get_param('account.autopost_batch_size')) or 1000
+        except ValueError as e:
+            batch_size = 1000
+            _logger.warning("Auto-post journal entry take default batch size because of the wrong value set in system parameter for key 'account.autopost_batch_size'")
```

Comment:
No need for a try except and a logger etc, just use the default value of 1000. 
If you can't manage batches of 1000, you have an issue elsewhere. (see the benchmark done above, you should be able to do 100k entries with standard code)
And trying to manage more than 1000 might be useless since it might get get in batches of 1000 by the ORM anyways. (just grep `split_for_in_conditions` in the code and see that it is used multiple times in `models.py`)

Reply:
I do this because I think if the multi invoice with too many lines then the user has the option to change this.
But ok I can also understand your point. so I removed this.

Reply:
The number of lines should not impact the posting of the journal entry. Except for invoices where the `invoice_date` is missing, but this is not a case where you should use `auto_post`, and we are still really within the lines.

Reply:
ok got it. 👍